File: output/gemma4-e2b/avatar/Java/Python/codeforces_581_A.py
import sys

def solve():
    # Read a and b from standard input
    try:
        # Read the line and split into two integers
        line = sys.stdin.read().split()
        if len(line) < 2:
            return
        a = int(line[0])
        b = int(line[1])
    except:
        # Handle potential empty input or formatting errors gracefully
        return

    count = 0
    # The loop runs as long as both a and b are positive
    while a > 0 and b > 0:
        count += 1
        a -= 1
        b -= 1

    # Calculate the result using the final values of a and b
    # Python's division operator (/) results in a float, matching the Java behavior.
    ans = a / 2 + b / 2

    # Print the result
    print(count, ans)

solve()

File: output/gemma4-e2b/avatar/Python/Java/codeforces_581_A.java
import java.util.Scanner;

public class codeforces_581_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        // Read the input line (e.g., "10 20")
        if (scanner.hasNextLine()) {
            String line = scanner.nextLine();
            String[] parts = line.split(" ");
            
            if (parts.length == 2) {
                try {
                    int a = Integer.parseInt(parts[0]);
                    int b = Integer.parseInt(parts[1]);
                    
                    int minVal = Math.min(a, b);
                    int maxVal = Math.max(a, b);
                    
                    // Calculate (max - min) // 2
                    int result = (maxVal - minVal) / 2;
                    
                    // Print min and the calculated value
                    System.out.println(minVal + " " + result);
                } catch (NumberFormatException e) {
                    // Handle case where input parts are not valid integers
                    // In a strict translation context, we might skip this, but for robustness:
                }
            }
        }
        scanner.close();
    }
}